src/reference/permissions.md

```diff
@@ -6,319 +6,67 @@ This section provides details about pre-configured permission tokens in Iroha 2.
-// that allows burning `alice_xor_id` asset
-let permission_token_to_alice: PermissionToken =
-    burn::CanBurnUserAssets::new(alice_xor_id).into();
-
-// Create an instruction that grants Bob permission to burn `alice_xor_id` asset
-let grant = Instruction::Grant(GrantBox::new(
-    permission_token_to_alice,
-    IdBox::AccountId(bob_id),
-));
-```
-
-### `CanUnregisterAssetWithDefinition`
-
-With `CanUnregisterAssetWithDefinition` permission token, a user can
-unregister assets with the corresponding asset definition.
-
-```rust
-let alice_id = AccountId::from_str("alice@test")?;
-let bob_id = AccountId::from_str("bob@test")?;
-let xor_id = AssetDefinitionId::from_str("xor#test")?;
-
-// Create a new `CanUnregisterAssetWithDefinition` permission token
-// that allows unregistering `xor_id` asset
-let permission_token_to_alice: PermissionToken =
-    unregister::CanUnregisterAssetWithDefinition::new(xor_id).into();
-
-// Create an instruction that grants Bob permission to unregister `xor_id` asset
-let grant = Instruction::Grant(GrantBox {
-    permission_token_to_alice.into(),
-    IdBox::AccountId(bob_id).into(),
-});
-```
-
-### `CanTransferUserAssets`
-
-With `CanTransferUserAssets` permission token, a user can transfer the
-specified asset.
-
-```rust
-let alice_id = AccountId::from_str("alice@test")?;
-let bob_id = AccountId::from_str("bob@test")?;
-let alice_xor_id = AssetId::new(
-    AssetDefinitionId::from_str("xor#test")?,
-    AccountId::from_str("alice@test")?,
-);
-
-// Create a new `CanTransferUserAssets` permission token
-// that allows to transfer `alice_xor_id` asset
-let permission_token_to_alice: PermissionToken =
-    transfer::CanTransferUserAssets::new(alice_xor_id).into();
-
-// Create an instruction that grants Bob permission to transfer `alice_xor_id` asset
-let grant = Instruction::Grant(GrantBox::new(
-    permission_token_to_alice,
-    IdBox::AccountId(bob_id),
-));
-```
-
-### `CanTransferOnlyFixedNumberOfTimesPerPeriod`
-
-With `CanTransferOnlyFixedNumberOfTimesPerPeriod` permission token, a user
-can transfer assets only a fixed number of times per some time period.
-
-```rust
-const PERIOD_MS: u64 = 5000;
-const COUNT: u32 = 2;
-
-// Create a new `CanTransferOnlyFixedNumberOfTimesPerPeriod` permission token
-// that limits the number of transfer to 2 per 5000 ms.
-let permission_token =
-    transfer::CanTransferOnlyFixedNumberOfTimesPerPeriod::new(PERIOD_MS.into(), COUNT);
-```
+Important!!!
+By default, all assets and accounts defined in the genesis block configuration file, are created by genesis@genesis account.
+That means "alice@wonderland" is not the owner of "rose#wonderland" and cannot grant permission for "rose#wonderland".
+To avoid this you can:
+1. Edit the genesis.json, add there a creation of "alice@wonderland" only and redeploy the Iroha 2.
+2. In the name of "alice@wonderland" create a subject (asset, definition, account etc.) and then give a permission for this.
```

Comment:
```suggestion
::: note
By default, all assets and accounts defined in the genesis block configuration file are created by `genesis@genesis` account. This means that `alice@wonderland` is not the owner of `rose#wonderland` and cannot grant permission for `rose#wonderland`.

To avoid this you can:
1. Edit the `genesis.json` file to only include the creation of  `alice@wonderland`, and then redeploy Iroha 2.
2. In the name of `alice@wonderland` create a subject (asset, definition, account etc.) and then give a permission for this.
:::
```

I've suggested changes but please make sure the meaning is preserved. Also, the item 2 is not clear to me, so it's probably needs rephrasing.

Another question, are 1 and 2 alternatives or consecutive steps?